src/inverted-index.js

```diff
@@ -104,6 +107,9 @@ class InvertedIndex {
   }
 
   /**
+   * Search
+   * Helps the search method to do the actual search
```

Comment:
### Nitpick

Remember the structure for doc blocks and how they should have line spacing? between title, description and params/return. We already talked about this.